framework/packages/abstract-standalone/src/state.rs

```diff
@@ -0,0 +1,80 @@
+use abstract_sdk::{
+    base::{ModuleId, ModuleMetadata, VersionString},
+    features::ModuleIdentification,
+    namespaces::{ADMIN_NAMESPACE, BASE_STATE},
+};
+use abstract_std::{
+    objects::{dependency::StaticDependency, module::ModuleInfo, nested_admin::NestedAdmin},
+    standalone::StandaloneState,
+    AbstractResult,
+};
+use cosmwasm_std::{StdResult, Storage};
+use cw_storage_plus::Item;
+
+/// The state variables for our StandaloneContract.
+pub struct StandaloneContract {
```

Comment:
I think it would be great to have dependencies on standalones as well. That way we can include the ibc client as a dependency, as well as enable multiple standalones to depend on each other (like a classical SC application)